integrations/jfrog.rst

```diff
@@ -0,0 +1,74 @@
+.. _integrations_jfrog:
+
+|jfrog_logo| JFrog
+==================
+
+Artifactory Build Info
+----------------------
+
+.. warning::
+
+    The support of Artifactory Build Info via extension commands is not guaranteed as stable yet.
+    See :ref:`the Conan stability<stability>` section for more information.
```

Comment:
The stability commitment cannot be referenced in this way, because it only applies to the Conan client, not to extensions. The warning should exclude it, not say it is not "stable" yet, it will never be "stable" according the Conan 2.0 stability committement, because it doesn't belong to the Conan 2.0 release

Reply:
changed